dynamiq/flows/base.py

```diff
@@ -65,7 +69,7 @@ def run_on_flow_start(
         """
         if config and config.callbacks:
             for callback in config.callbacks:
-                callback.on_flow_start(self.model_dump(), input_data, **kwargs)
+                callback.on_flow_start(self.to_dict(for_tracing=True), input_data, **kwargs)
```

Comment:
Please use `for_tracing=True` just for Tracing callback, not for all. The same for other callbacks callers